### DISCUSSION TITLE
Query on Specifying HEAT_OF_COMBUSTION in &REAC vs &MATL for Vegetation Burning Simulations

### DISCUSSION THREAD

**Discussion by Sidfds** [firemodels/fds#13449] [Q&A]
Hi everyone,

I have a query regarding the difference between specifying HEAT_OF_COMBUSTION in the &REAC line versus specifying it in the &MATL ID line. My main interest is ensuring accurate heat release rates and mass loss measurements while simulating vegetation burning.

Thank you.

**Comment by TristanHehnen** [comment]
In the `REAC` line, the HOC of the specific combustion reaction of your chosen surrogate fuel is defined.

In the `MATL`, the HOC is used to scale the surrogate fuel release such that a desired heat release is met. 

Depending on how you set up you model, you could make sure that your surrogate fuel has the heat of combustion you would like, then mass loss and heat release from your solid match. 
Keep in mind that mass loss does not necessarily equal heat release. Consider wood that may release water vapour around 100°C. This would be mass loss that does not contribute to HRR.
